calliope/backend/pyomo/constraints/group.py

```diff
@@ -79,3 +91,52 @@ def demand_share_constraint_rule(backend_model, constraint_group, carrier, what)
         )
 
         return equalizer(lhs, rhs, what)
+
+
+def supply_share_constraint_rule(backend_model, constraint_group, carrier, what):
+    """
+    Enforces shares of carrier_prod for groups of technologies and locations. The
+    share is relative to ``supply`` and ``supply_plus`` technologies only.
+
+    .. container:: scrolling-wrapper
+
+        .. math::
+
+            \\sum_{loc::tech::carrier \\in given\\_group, timestep \\in timesteps} carrier_{prod}(loc::tech::carrier, timestep) <= / >=
```

Comment:
Can I build that locally? I have no idea of the syntax and whether this is correct.

Reply:
If you build the docs (`build html` when in `calliope/docs`) then it should be possible to see the result of that in the locally built html files. Needs a bunch of dependencies installed to build the docs (see the relevant requirements file)

Reply:
cheers, I'll give it a try